aksel.nav.no/website/pages/god-praksis/[slug].tsx

```diff
@@ -67,7 +68,7 @@ export const getServerSideProps: GetServerSideProps = async (
       articles: await Promise.all(
         articles.map(async (a) => ({
           ...a,
-          publishedAt: await dateStr(a.publishedAt),
+          lastVerified: await dateStr(a.lastVerified ?? a.publishedAt),
```

Comment:
We could consider excluding `publishedAt` from the payload if we don't need it after this, unless it's more trouble than it's worth.

Reply:
Remove now ⚰️

Reply:
Wait, don't we need `publishedAt` as fallback when `lastVerified` is undefined?

Reply:
Thought so at first myself, thats why i left it at first, but then remembered that both of those values are undefined until first publish where both are created. It shouldn't be possible to publish an article in sanity without lastVerified being defined 🤞